src/figuresClasses.ts

```diff
@@ -1,19 +1,77 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
+type GetArea = () => number;
 
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea: GetArea;
```

Comment:
You can wtire it sorten
```suggestion
export interface Figure {
  shape: Shape;
  color: Color;
  getArea(): number;
```